apps/editors/templates/editors/review.html

```diff
@@ -163,7 +163,7 @@ <h3 id="history">
       </div>
 
       <div class="review-actions-section review-actions-files data-toggle"
-           data-value="{{ actions_minimal|join("|") }}"{% if allow_unchecking_files %} data-uncheckable="1"{% endif %}>
+           data-value="{{ actions_minimal|join("|") }}" data-uncheckable="1">
```

Comment:
You may as well just get rid of the attribute and the JS code that relies on it.